sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -349,10 +342,9 @@ keyAsyncClient.getKey("keyName").subscribe(keyResponse -> {
 
```

Comment:
[here](https://github.com/Azure/azure-sdk-for-java/pull/6061/files#diff-05d7c287ef67008ffcb79af0816ba946L338), it should be updated to `getKeyProperties`

Reply:
you mean updateKeyProperties, I assume
Updated